vicky/src/lib/database/schema.rs

```diff
@@ -16,9 +16,9 @@ diesel::table! {
         id -> Uuid,
         display_name -> Varchar,
         status -> Varchar,
-        features -> Array<Nullable<Text>>,
+        features -> Array<Text>,
```

Comment:
We can assume, that this does not need a migration?

Reply:
It does not. I don't even know why diesel does this stuff in the first place..

Reply:
If you can put nullable places in the array, that'd be weird because the migration explicitly states NON NULL